src/components/Dialogs/Conditions/OrgUnitCondition.js

```diff
@@ -73,6 +76,102 @@ const OrgUnitCondition = ({
     )
 }
 
+// VERSION-TOGGLE: rename to OrgUnitCondition when 42 is lowest supported version
+const OrgUnitConditionMinVersionV42 = ({
+    addMetadata,
+    addParentGraphMap,
+    condition,
+    onChange,
+    metadata,
+    parentGraphMap,
+}) => {
+    const { rootOrgUnits, currentUser } = useCachedDataQuery()
+    const selected = useMemo(() => {
+        const idsString = condition.split(':')?.[1] ?? ''
+
+        if (idsString.length === 0) {
+            return []
+        }
+
+        return idsString.split(';').map((id) => {
+            const item = {
+                id,
+                name: metadata[id].displayName || metadata[id].name,
+            }
+            const isOuTreeItem =
+                !id.startsWith('LEVEL') &&
+                !id.startsWith('OU_GROUP') &&
+                !id.startsWith('USER_ORGUNIT')
+
+            if (isOuTreeItem) {
+                item.path = getOuPath(id, metadata, parentGraphMap)
+            }
+
+            return item
+        })
+    }, [condition, metadata, parentGraphMap])
+    const onSelect = useCallback(
```

Comment:
silly little comment, but could you add a newline, just for some breathing room in the file?